src/Umbraco.Forms.Integrations.Crm.Hubspot/Services/HubspotContactService.cs

```diff
@@ -21,14 +22,14 @@ namespace Umbraco.Forms.Integrations.Crm.Hubspot.Services
 {
     public class HubspotContactService : IContactService
     {
-        // Using a static HttpClient (see: https://www.aspnetmonsters.com/2016/08/2016-08-27-httpclientwrong/).
+        //Using a static HttpClient(see: https://www.aspnetmonsters.com/2016/08/2016-08-27-httpclientwrong/).
         private static readonly HttpClient s_client = new HttpClient();
 
-        // Access to the client within the class is via ClientFactory(), allowing us to mock the responses in tests.
-        internal static Func<HttpClient> ClientFactory = () => s_client;
+        //Access to the client within the class is via ClientFactory(), allowing us to mock the responses in tests.
+        public static Func<HttpClient> ClientFactory = () => s_client;
 
-        private readonly IFacadeConfiguration _configuration;
-        private readonly ILogger _logger;
+        private readonly IConfiguration _configuration;
```

Comment:
A better pattern here I think is rather than injecting `IConfiguration`, we create typed options we can access via `IOptions`.

Here's an example from Forms:

- [Typed representations of configuration](https://github.com/umbraco/Forms/tree/v10/dev/src/Umbraco.Forms.Core/Configuration).
- [Registering them](https://github.com/umbraco/Forms/blob/v10/dev/src/Umbraco.Forms.Core/Extensions/UmbracoBuilderExtensions.cs#L154),
- [Using the config](https://github.com/umbraco/Forms/blob/v10/dev/src/Umbraco.Forms.Core/Services/WorkflowExecutionService.cs#L56).